wasm_runtime/src/c_api.rs

```diff
@@ -468,3 +468,162 @@ pub extern "C" fn wasm_executionctx_run(executionctx_id: *const c_char, _buffer:
 pub extern "C" fn wasm_return_const_char_ownership(ptr: *const c_char) {
     deallocate_cstring(ptr);
 }
+
+
+// ##########################################################################
+//                              Unit Tests
+// ##########################################################################
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::ffi::CString;
+    use crate::config::WASM_RUNTIME_CONFIGS;
+
+    #[test]
+    fn wasm_wasm_module_load() {
```

Comment:
Just to keep coherence with other test cases naming convention:

```suggestion
    fn wasm_module_load_existing_nonexisting() {
```